app/services/auth.service.js

```diff
@@ -120,6 +121,10 @@ function initialise (app) {
   passport.use('localDirect', new CustomStrategy(localDirectStrategy))
   passport.serializeUser(serializeUser)
   passport.deserializeUser(deserializeUser)
+  app.use((req, res, next) => { 
+    req.user && addUserLoggingFields(req.user) 
```

Comment:
(very minor!) I'm a fan of short circuit evaluation for assignment but have seen it get a bit confusing to read when used for method calls that have side effects. 

Interested in your opinion but I think these would benefit from being a bit more verbose but a bit clearer 
```suggestion
    if (req.user) {
      addUserLoggingFields(req.user) 
    }
```